syntax = "proto3";

package rpc_general;

enum StatusEnum {
    STATUS_SUCCESS = 0;
    STATUS_ERROR = 1;
    STATUS_BUSY = 2;
    STATUS_TIMEOUT = 3;
    STATUS_UNSUPPORTED = 4;
}

message Status {
    StatusEnum status = 1;
}

